taiga2/controllers/endpoint.py

```diff
@@ -1242,7 +1322,7 @@ def add_figshare_token(token: str):
 
 def _fetch_figshare_token() -> str:
```

Comment:
It looks like you moved this function to `taiga2/third_party_clients/figshare.py` If that's the case, can you delete this instance of it?